azurerm/internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -202,6 +243,9 @@ func resourceArmLogAnalyticsWorkspaceRead(d *schema.ResourceData, meta interface
 		d.Set("location", azure.NormalizeLocation(*location))
 	}
 
+	d.Set("public_network_access_for_ingestion_enabled", resp.PublicNetworkAccessForIngestion != operationalinsights.Disabled)
+	d.Set("public_network_access_for_query_enabled", resp.PublicNetworkAccessForQuery != operationalinsights.Disabled)
```

Comment:
The Azure API can omit this field from the response for longer resources / resources which haven't been modified - as such assuming this is enabled by default is a security risk:

```suggestion
	d.Set("public_network_access_for_query_enabled", resp.PublicNetworkAccessForQuery == operationalinsights.Enabled)
```

Reply:
will change